classes/game.rb

```diff
@@ -0,0 +1,29 @@
+require 'date'
+require_relative './item'
+
+class Game < Item
+  attr_accessor :publish_date, :last_played, :multiplayer
+  attr_reader :id
+
+  def initialize(publish_date, last_played, multiplayer)
+    super(publish_date)
+    @last_played = last_played
+    @multiplayer = multiplayer
+    @archived = can_be_archived?
+  end
+
+  def as_hash
+    {
+      'multiplayer' => @multiplayer,
+      'last_played' => @last_played,
+      'publish_date' => @publish_date,
+      'archived' => @archived
+    }
+  end
+
+  private
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/195665264-36d48ba9-e670-41ca-ade1-bc1290879b57.png)

- [ ] When i try to add a game it shows the error shown in the image :arrow_up: Kindly fix this error :+1:

Reply:
@HAMMAS-SALEEM this data need to be yes or not because is booleam

Reply:
Hey @nimplay you have said right. My apologies for my mistake. You are free to go. I hope next reviewer will approve your project :blush:

Reply:
Many thanks for the comment.
Corrected by adding data validation method.